ci/ansible/pulp_server.yaml

```diff
@@ -6,4 +6,5 @@
       pulp_install_server: true
     - role: lazy
       when: pulp_version|float >= 2.8
+    - role: pulp_certs
```

Comment:
Do you mind renaming this to pulp-certs? This is the style I've been following for the other role names.